core/templates/dev/head/css/oppia.css

```diff
@@ -253,6 +254,26 @@ textarea {
 }
 
 /*
+  Styles for the site feedback form.
+*/
+.oppia-site-feedback {
```

Comment:
I recommend this CSS:

  background-color: #fff;
  bottom: 4px;
  color: rgba(0,0,0,0.87);
  cursor: pointer;
  left: 4px;
  opacity: 0.5;
  padding: 4px 8px;
  position: fixed;
  z-index: 5;